doc/BuiltInMetrics.md

```diff
@@ -21,25 +21,27 @@ instance_number | A Guid representing the current runtime. On restart, all metri
+| `edgehub_direct_methods_total`                              |  `from` (Message source)<br> `to` (Message destination)														| Total number of direct messages sent										| counter |
+| `edgehub_queue_length`                                      |  `endpoint` (Message source)<br> `priority` (queue priority)												| Current length of edgeHub's queue for a given priority               		| gauge |
+| `edgehub_messages_dropped_total`                            |  `reason` (no_route, ttl_expiry)<br> `from` (Message source)<br> `from_route_output` (Output that sent the message)<br>	| Total number of messages removed because of reason			| counter |
+| `edgehub_messages_unack_total`                              |  `reason` (storage_failure)<br> `from` (Message source)<br> `from_route_output` (Output that sent the message)<br>	| Total number of messages unack because storage failure			| counter |
+| `edgehub_offline_count_total`                               |  `id` (Module ID)<br>                                                      									| Total number of times edgeHub went offline								| counter | 
+| `edgehub_offline_duration_seconds`                          |  `id` (Module ID)<br> `quantile`(Percentile [50, 90, 95, 99, 99.9, 99.99]) 									| P50, P90, P95, P99, P99.9 and P99.99 time edge hub was offline. Values may be reported as `NaN`  if no new measurements are reported for a  certain period of time (currently 10 minutes). As this is `summary` type, corresponding `_count` and `_sum` counters will be emitted.              	| summary |
+| `edgehub_operation_retry_total`      						  |  `id` (Module ID)<br>`operation` (Operation name)                                                        	| Total number of times edgeHub operations were retried 					| counter | 
+| `edgehub_client_connect_failed_total`                       | `id` (Device/Module ID)<br> `reason` (not authenticated)<br> 												| Total number of times each client failed to connect to edgeHub		| counter |                                                              
+| `edgehub_connected_clients`                              	  | 															 												| Current number of clients connected to edgeHub        					| gauge |                                                            
+| `edgehub_client_connect_success_total`                      | `id` (Device/Module ID)<br>																					| Total number of times each client successfully connected to edgeHub   | counter |                                                                                                                                                                    | counter |                                                              
```

Comment:
1. With the current implementation, there is no distinction between have the same module connected & reconnected multiple times vs having multiple modules connect at the same time. I'm not sure what the customer use case is, but I'm having a hard time interpreting the meaning of this metric.
2. This metric only accurate per edgeHub restart cycle.

Reply:
1. There is `id` in the metrics which will tell which device/module it is. The counter will group based on the `id`.
2. Ideally metrics should be collected periodically. In user scenario if customer  would like to monitor edge hub, the metrics should be feed to Azure monitoring or tracking or ELK search.  Restart causes metrics data rest shouldn't be considered in the implementation unless we'd like to persist it locally.